src/tx-utils.js

```diff
@@ -6,14 +6,15 @@ import { feeFactors } from './fee-factors.js'
 import { bufToHex } from './utils.js'
 import { fix2Float, float2Fix, floorFix2Float } from './float16.js'
 import { getPoolTransactions } from './tx-pool.js'
-import { getAccountIndex } from './addresses.js'
+import { getAccountIndex, getEthereumAddress, isHermezEthereumAddress, isHermezAccountIndex } from './addresses.js'
 import { getAccount } from './api.js'
 import { getProvider } from './providers.js'
 
 export const TxType = {
   Deposit: 'Deposit',
   CreateAccountDeposit: 'CreateAccountDeposit',
   Transfer: 'Transfer',
+  TransferEthAddr: 'TransferToEthAddr',
```

Comment:
What do you think about change this name to `TransferToEthAddr`. Because when reading it seems like you are transfering an ethereum address haha